genanki/__init__.py

```diff
@@ -331,7 +331,7 @@ def write_to_file(self, file):
       outzip.writestr('media', json.dumps(media_json))
 
       for i, f in media_json.items():
-        outzip.write(f, str(i))
+        outzip.write(media_path + f, str(i))
```

Comment:
> just put the full path in the media files list.

I've tried it before I've done some changes. It doesn't work like that. When you set the full file path in media list, apkg creates totally fine, but media files don't copy when you import it. I think it's because media file in apkg looks like that:
![image](https://user-images.githubusercontent.com/19855693/49338985-60108980-f63b-11e8-9996-3570b6b29d48.png)